Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "winit"
-version = "0.7.2"
+version = "0.7.3"
```

Comment:
I think bumping this is usually left for the maintainer to do immediately prior to making a release.